website/views/shared/header/header.jade

```diff
@@ -12,7 +12,7 @@
       .meter.health(tooltip='{{Math.round(user.stats.hp * 100) / 100}}')
         .bar(ng-style='{"width": Shared.percent(user.stats.hp, 50)+"%"}')
         span.meter-text.value
-          | {{Math.ceil(user.stats.hp)}} / 50
+          | {{Math.ceil(user.stats.hp)}} / Shared.maxHealth
```

Comment:
This will render the text "Shared.maxHealth" instead of "50". You need to wrap it in double curly braces: `{{Shared.maxHealth}}` since it's referencing: https://github.com/HabitRPG/habitrpg/blob/develop/website/public/js/controllers/rootCtrl.js#L30

In fact, since it's a static variable, I'd add the one-time binding symbols to it: `{{:Shared.maxHealth}}`